### PR TITLE
Case list actions speed improvement

### PR DESCRIPTION
**Pull request by maarten-ritense**
Add query param canInitializeDocument to ProcessDocumentResource#findProcessDocumentDefinitions to be able to filter on it

### Breaking changes

- [X] The contribution only contains changes that are not breaking.

### Documentation

- [x] Not applicable

Link to the pull request in the [Valtimo documentation repository](https://github.com/valtimo-platform/valtimo-documentation):
>

New features or changes that have been introduced have been documented.
- [X] Not applicable

### Tests
Unit tests have been added that cover these changes
- [ ] Yes
- [ ] Not applicable

Integration tests have been added that cover these changes
- [X] Yes

### Security
The [Secure by Default principle](https://en.wikipedia.org/wiki/Secure_by_default) has been applied to these changes
- [X] Not applicable

Added or changed REST API endpoints have authentication and authorization in place
- [X] Not applicable

Valtimo access control checks have been implemented
- [X] Not applicable

### Dependencies
Newly added dependencies do not introduce known vulnerabilities/CVE's and are in line with the Valtimo license
- [X] Not applicable

### CODE REVIEW

File: zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt [hunk 1]
```diff
@@ -111,7 +111,7 @@ class ZaakTypeLinkServiceTest {
 
     @Test
     fun `should get empty zaakTypeLinks`() {
-        whenever(processDocumentAssociationService.findProcessDocumentDefinitions("documentDefinitionName", true))
+        whenever(processDocumentAssociationService.findProcessDocumentDefinitions("documentDefinitionName", null, null))
```
Comment (ivo-ritense) [inline zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt]: The fact that this is needed means the method is breaking, right? Maybe we should overload this?

Comment (maarten-ritense) [reply zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt]: yes, maybe that is true. I will have a look

Comment (maarten-ritense) [reply zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt]: @ivo-ritense suggestions are welcome :-)

Comment (maarten-ritense) [reply zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt]: @ivo-ritense I removed the mock as it does not seem to be needed/used by this specific unit test

**Review by marijnritense — APPROVED** [review]

**Pull request merged**